code/game/area/Space_Station_13_areas.dm

```diff
@@ -1513,6 +1524,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Telecomms Control Room"
 	icon_state = "tcom_sat_comp"
 	sound_environment = SOUND_AREA_MEDIUM_SOFTFLOOR
+	mood_job_allowed = list(JOB_NAME_RESEARCHDIRECTOR, JOB_NAME_SCIENTIST, JOB_NAME_ROBOTICIST)  // techies
+	mood_bonus = 2
+	mood_message = "<span class='nicegreen'>All this technology is so fascinating, I could sit here for hours..\n</span>"
```

Comment:
I feel like anyone with a tech job would rather go outside than stare at technology

Reply:
as a tech person i dont go outside so no

Reply:
just change this to engineers and CE and I'd be okay with it. RD and scientists should not be in tcomms.